products/kuma.md

```diff
@@ -0,0 +1,51 @@
+---
+title: kuma
+category: app
+tags: cncf
+permalink: /kuma
+alternate_urls:
+-   /kuma
+changelogTemplate: https://github.com/kumahq/kuma/releases/tag/__LATEST__
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+  methods:
+  -   git: https://github.com/kumahq/kuma.git
+      regex: ^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(-ce)?$
```

Comment:
```suggestion
  -   custom: kuma
```

Tags are formatted `x.y.z`, so the default regex already works. Also added the custom script added in https://github.com/endoflife-date/release-data/pull/359.